server/middleware/setUpSentry.ts

```diff
@@ -21,6 +21,12 @@ function setUpSentryRequestHandler(app: express.Express): void {
       dsn: config.sentry.dsn,
       environment: config.environment,
       release: applicationVersion.gitRef,
+      integrations: [
+        // enable HTTP calls tracing
+        new Sentry.Integrations.Http({ tracing: true }),
+        // enable Express.js middleware tracing
+        new Sentry.Integrations.Express({ app }),
+      ],
```

Comment:
Presumably this tracing doesn't undermine the anonymisation that's motivated it?

Reply:
No, the way it usually works is it assigns an id to an individual request so that request can be traced through multiple services. It'll still be an anonymous ID.